test/mounted_files_test.py

```diff
@@ -105,18 +105,13 @@ def test_mounted_files_serialized(supports_dir, env_mount_files):
     }
 
 
-def test_mounted_files_package(supports_dir, env_mount_files):
-    p = subprocess.run([sys.executable, "-m", "pkg_a.package"], cwd=supports_dir, capture_output=True)
+def test_mounted_files_package(supports_dir, env_mount_files, servicer, server_url_env):
+    p = subprocess.run(["modal", "run", "pkg_a.package"], cwd=supports_dir, capture_output=True)
     assert p.returncode == 0
-    stdout = p.stdout.decode("utf-8")
-    stderr = p.stderr.decode("utf-8")
-    print("stdout: ", stdout)
-    print("stderr: ", stderr)
-    files = set(stdout.splitlines()).difference(env_mount_files)
 
+    files = set(servicer.files_name2sha.keys()) - set(env_mount_files)
     # Assert we include everything from `pkg_a` and `pkg_b` but not `pkg_c`:
     assert files == {
-        "/root/package.py",
```

Comment:
This file wasn't actually being mounted like this in production, probably due to some deduplication, but appeared in this test due to the test only using some parts of the mounting code. Refactoring the test to actually run the mount code e2e and checking what's being sent over the wire let me remove this incorrect assertion.